packages/smooth_app/lib/pages/scan/mkit_scan_helper.dart

```diff
@@ -233,13 +233,23 @@ class _MLKitScanDecoderIsolate {
 
     port.send(
       barcodes
-          .map((Barcode barcode) => barcode.rawValue)
+          .map((Barcode barcode) => _changeBarcodeType(barcode))
           .where((String? barcode) => barcode?.isNotEmpty == true)
           .cast<String>()
           .toList(growable: false),
     );
   }
 
+  static String? _changeBarcodeType(Barcode barcode) {
+    //EAN13 begins with 0 is detected as UPC-A by google_ml_barcode_scanner v0.0.2
```

Comment:
The library has changed, please name the new one called `google_mlkit_barcode_scanning`

Reply:
> The library has changed, please name the new one called `google_mlkit_barcode_scanning`

✅